include/c_types/contracted_rt.h

```diff
@@ -44,6 +44,8 @@ struct contracted_rt {
     int64_t source;
     int64_t target;
     double cost;
+    int64_t vertex_order;
+    int64_t metric;
```

Comment:
Move that in the corresponding commit in #6?

Reply:
I keep it there, since I decided to remove all traces of the new contraction method in #6